frontend/app/src/components/ui/form.tsx

```diff
@@ -47,6 +59,10 @@ export const Form = React.forwardRef<FormRef, FormProps>(
               event.stopPropagation();
             }
 
+            if (slideOverContext?.setPreventClose) {
+              slideOverContext?.setPreventClose(false);
+            }
+
```

Comment:
if we submit and the submit fails validation, we are able to close the drawer again with unsaved changes